src/Psalm/Checker/FunctionChecker.php

```diff
@@ -602,24 +614,28 @@ public static function getFQFunctionNameFromString($function_name, StatementsSou
             return substr($function_name, 1);
         }
 
+        $function_name_lcase = strtolower($function_name);
+
         $imported_function_namespaces = $source->getAliasedFunctions();
         $imported_namespaces = $source->getAliasedClasses();
 
         if (strpos($function_name, '\\') !== false) {
             $function_name_parts = explode('\\', $function_name);
             $first_namespace = array_shift($function_name_parts);
+            $first_namespace_lcase = strtolower($first_namespace);
 
-            if (isset($imported_namespaces[strtolower($first_namespace)])) {
-                return $imported_namespaces[strtolower($first_namespace)] . '\\' . implode('\\', $function_name_parts);
+            if (isset($imported_namespaces[$first_namespace_lcase])) {
+                return $imported_namespaces[$first_namespace_lcase] . '\\' . implode('\\', $function_name_parts);
             }
 
-            if (isset($imported_function_namespaces[strtolower($first_namespace)])) {
-                return $imported_function_namespaces[strtolower($first_namespace)] . '\\' . implode('\\', $function_name_parts);
+            if (isset($imported_function_namespaces[$first_namespace_lcase])) {
+                return $imported_function_namespaces[$first_namespace_lcase] . '\\' .
+                    implode('\\', $function_name_parts);
             }
-        } elseif (isset($imported_namespaces[strtolower($function_name)])) {
-            return $imported_namespaces[strtolower($function_name)];
-        } elseif (isset($imported_function_namespaces[strtolower($function_name)])) {
-            return $imported_function_namespaces[strtolower($function_name)];
+        } elseif (isset($imported_namespaces[$function_name_lcase])) {
```

Comment:
These weren't a PSR-2 thing, but it just seemed wasteful to be running `strtolower` every time here.